web/template/payments.gohtml

```diff
@@ -15,9 +15,45 @@
             {{ else }}
                 <table class="govuk-table my-table table__no-border">
                     <tbody class="govuk-table__body">
+                    <tr class="govuk-table__row">
+                        {{ if gt .OutstandingFee 0 }}
+                            <th scope="row" class="govuk-table__header">
+                                <h2 class="govuk-heading-m govuk-!-padding-top-3 govuk-!-padding-bottom-2">Outstanding fee due: </h2>
+                            </th>
+                            <td class="govuk-table__cell">
+                                <h2 class="govuk-heading-m govuk-!-padding-top-3">£{{ fee .OutstandingFee }}</h2>
+                            </td>
+                        {{else if .RefundAmount }}
+                            <th scope="row" class="govuk-table__header">
+                                <h2 class="govuk-heading-m govuk-!-padding-top-3 govuk-!-padding-bottom-2">Refund due: </h2>
+                            </th>
+                            <td class="govuk-table__cell">
+                                <h2 class="govuk-heading-m govuk-!-padding-top-3">£{{ fee .RefundAmount }}</h2>
+                            </td>
+                        {{ else }}
+                            <th scope="row" class="govuk-table__header">
+                                <h2 class="govuk-heading-m govuk-!-padding-top-3 govuk-!-padding-bottom-2">No further fees due</h2>
+                            </th>
+                            <td class="govuk-table__cell">{{/*required for table styling*/}}</td>
```

Comment:
You can get over the empty cell problem with `colspan="2"`, but also this should be a `td` not a `th` because it's not acting as the header of anything
```suggestion
                            <td class="govuk-table__cell govuk-heading-m govuk-!-padding-top-3 govuk-!-padding-bottom-2" colspan="2">No further fees due</td>
```

Reply:
I've amended those now `colspan` only worked if i removed `govuk-heading-m` as it was setting `display` to `block` and overriding the `table-cell `setting. I also added a css class to override the border colour so it is black and not dark grey for the first headings. I've ran this by claire and she has approved